managed-databases/postgresql-and-mysql/index.mdx

```diff
@@ -1,8 +1,87 @@
 ---
 meta:
-  title: Managed Database for PostgreSQL and MySQL Documentation
-  description: Databases Documentation
-content:
-  h1: Managed Database for PostgreSQL and MySQL Documentation
-  paragraph: Databases Documentation
+  title: Managed Database for PostgreSQL and MySQL
+  description: Dive into Scaleway Managed Database for PostgreSQL and MySQL with our quickstart guides, how-tos, tutorials and more.
 ---
+
+<ProductHeader
+   productName="Managed Database for PostgreSQL and MySQL"
+   productLogo="managedDatabaseWire"
```

Comment:
I don't see an image on the banner when I do a local build, do you?

Edit, actually this is the same for all the product pages in this PR? Not sure why...

Reply:
I think it's because you need to take out the "Wire" at the end of the product name in productLogo - to check

Reply:
```suggestion
   productLogo="managedDatabase"
```

Reply:
Fyi: the names of all available product logos are here https://www.scaleway.com/en/docs/components/#ultraviolet-illustrations